Sources/JNI/JavaParameterConvertible+Primitives.swift

```diff
@@ -107,3 +107,29 @@ extension Float: JavaParameterConvertible, JavaInitializableFromMethod, JavaInit
         return try jni.GetFloatField(of: javaObject, id: fieldID)
     }
 }
+
+// Int64 aka JavaLong
+
+extension Int64: JavaParameterConvertible, JavaInitializableFromMethod, JavaInitializableFromField {
```

Comment:
we should define this as an extension on `JavaLong`, and return `JavaLong` wherever relevant - it's the same type, but it makes the API's intent clearer